### DISCUSSION TITLE
pfsense +

### DISCUSSION THREAD

**Discussion by philippenc** [jaredhendrickson13/pfsense-api#415] [General]
Hello, with Netgate's policy change regarding the acquisition of non-commercial use tokens and the discontinuation of updates for those with a non-commercial use PFSENSE+, are you going to continue to develop the API for future versions of PFSENSE+?

**Comment by jaredhendrickson13** [comment]
Yes, I am still committed to supporting the package on pfSense Plus however I can. I am exploring some options to retain an automated test environment for pfSense Plus, but more realistically I will need to rely on the community to help support Plus going forward. I am planning on making an announcement this weekend with specifics.
